lisa/tools/chmod.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Chmod(Tool):
+    @property
+    def command(self) -> str:
+        return "chmod"
+
+    def update_folder_permission(
```

Comment:
remove "permission" from the name, the chmod is all about change the permission.